certbot/tests/util.py

```diff
@@ -276,6 +276,15 @@ def _write_msg(message, *unused_args, **unused_kwargs):
         """
         stdout.write(message)
 
+    def mock_method(*args, **kwargs):
+        """
+        Mock function for IDisplay methods.
+        """
+        message = args[0] if args else kwargs.get('message', None)
```

Comment:
I think you should be able to just call `_write_msg(*args, **kwargs)` and Python should properly handle assigning parameters to variables.